README.md

```diff
@@ -1,8 +1,9 @@
-# EctoJsonapi
+## Ecto. Json.  Api.
+Ecto => Json (Api)
+<br/>Convert an ecto schema into json.
 
-EctoJsonapi is tool for dealing with JsonAPI and Ecto schemas
-It:
-- Converts Ecto schemas into elixir maps structured like JsonApi v1.0 with
-  the `EctoJsonapi.Load/2`
-- Converts elixir maps that are structured like JsonApi v1.0 into maps
-  structured like Ecto schemas with `EctoJsonapi.Dump/1`
+## Nitty Gritty
+
+EctoJsonapi is tool for dealing with JsonAPI and Ecto schemas:
+1) `EctoJsonApi.Dump/1` Converts elixir maps Ecto schemas. 
```

Comment:
changed 
```
- Converts Ecto schemas into elixir maps structured like JsonApi v1.0 with
  the `EctoJsonapi.Load/2`
```
to 
```
1) `EctoJsonApi.Dump/1` Converts elixir maps Ecto schemas. 
```

@mrmicahcooper, can you read though this and make sure I am not mis-wording in the changes.

Reply:
Maybe 
```
- Converts Ecto schemas into elixir maps structured like JsonApi v1.0 with
  the `EctoJsonapi.Load/2`
```
can be broken into multiple list items?

Reply:
possibly: 
```
1) `EctoJsonApi.Dump/1` Converts elixir maps Ecto schemas. 
2) `EctoJsonApi.Dump/1` Converts JsonApi v1.0 syntax into Ecto schemas. 
```

Reply:
I think it's actually important to note the Jsonapi version as this may not work with newer breaking versions.

Reply:
Do you think the update in the second pr adequately solves the problem?
https://github.com/mrmicahcooper/ecto_jsonapi/pull/2/files#diff-04c6e90faac2675aa89e2176d2eec7d8R7

Reply:
```
EctoJsonapi is tool for dealing with JsonAPI and Ecto schemas:
1) `EctoJsonApi.Dump/1` Converts elixir maps Ecto schemas.
2) `EctoJsonApi.Dump/1` Converts JsonApi v1.0 syntax into Ecto schemas.
3) `EctoJsonApi.Load/2` Converts Ecto schemas into elixir maps structured like JsonApi v1.0.
```

Happy to have a rewritten draft from yourself.